website/blog/2024-04-05-release-1.9.md

```diff
@@ -0,0 +1,221 @@
+---
+title: Podman Desktop 1.9 Release
+description: Podman Desktop 1.9 has been released!
+slug: podman-desktop-release-1.9
+authors: benoitf
+tags: [podman-desktop, release, podman]
+hide_table_of_contents: false
+image: /img/blog/podman-desktop-release-1.9/juggling.png
+---
+
+import ReactPlayer from 'react-player'
+
+Podman Desktop 1.9 Release! 🎉
+
+![Podman-desktop-1-9-hero](img/podman-desktop-release-1.9/juggling.png)
+
+We've got a new release with a ton of seal appeal! This release introduces:
+
+- **Podman 5 !** [Podman 5.0.1](https://github.com/containers/podman/releases/tag/v5.0.1) as an experimental feature for existing users or set as default for new users.
+- **Podman 4.9.4**: [Podman 4.9.4](https://github.com/containers/podman/releases/tag/v4.9.4) is now included in both Windows and macOS installers.
+- **Backup/Restore Images**: Save images or containers to tar archives and restore them.
+- **Kubernetes Pods Terminal**: Connect to a terminal within Kubernetes pods.
+- **Extension API Improvements**: Additional updates to the extension API used by 🦭 Podman Desktop's extensions.
+
+Podman Desktop 1.9 is now available. [Click here to download it](/downloads)!
+
+<!--truncate-->
+
+---
+
+## Release Details
+
+### Podman v5 (5.0.1)
+
+🦭 Podman version 5 is now accessible as an experimental feature for existing Podman users and is set as the default for new users.
+
+Why experimental for existing users ? The main reason is that a Podman machine version 4 is not compliant at all with a Podman machine version 5.
+
+Moving to Podman v5 implies to optionally save images and then delete previous machines and create a new one.
+
+#### Important Steps before updating
+
+Prioritize data backup by using the save feature in the `Image Lists` section. This feature allows you to back up your images and restore them once you have a new Podman machine.
+
+When prompted to update, confirm to remove all existing data from your machines.
+
+For more details on the save/load feature, refer to the release notes.
+
+![v5 update](img/podman-desktop-release-1.9/podman-update-to-v5.png)
+
+**_NOTE:_** On macOS you will notice a lot of performance enhancements as Apple Hypervisor is used instead of QEMU for the Podman machine. Expect better performance on the file sharing.
```

Comment:
I think we should start the v5 section with this line and the next one - i.e. Podman v5 is out, it's awesome, comes with huge improvements for Mac - but is incompatible so we're starting the roll out for new users only as we test and improve migration.